src/main/webapp/main/Main.js

```diff
@@ -44,9 +44,9 @@ class Main extends Component {
 
   updateDetailsStatus = (status) => {
     this.setState(prev => {
-      prev.status.projectReviewApproved = status.projectReviewApproved;
-      prev.status.summary = status.summary;
-      prev.status.actor = status.pm;
+      prev.status.projectReviewApproved = status.extraProperties.projectReviewApproved;
+      prev.status.summary = status.issue.summary;
+      prev.status.actor = status.extraProperties.actor;
```

Comment:
I think there is a solid chance of some of these values being null, especially for legacy projects.

Reply:
Greg, I added validations to check null cases